index.bs

```diff
@@ -2680,25 +2693,19 @@ WritableStream(<var>underlyingSink</var> = {}, { <var>size</var>, <var>highWater
   behavior will be the same as a {{CountQueuingStrategy}} with a <a>high water mark</a> of 1.
 </div>
```

Comment:
It would be good to add the note about how the inFlightX slots are mutually exclusive with the X slots here (as a `<p class="note">`).